python/dask_cudf/dask_cudf/core.py

```diff
@@ -179,29 +179,6 @@ def set_index(self, other, **kwargs):
             )
         return super().set_index(other, shuffle="tasks", **kwargs)
 
-    def reset_index(self, force=False, drop=False):
-        """Reset index to range based
-        """
-        if force:
```

Comment:
@beckernick @randerzander -  Just curious: Was this behavior originally added by request from one of you guys? (to get a global 0->N index)

Reply:
Looks like we got the answer here off-line (this was not in use)